app/wizards/claims/add_school_wizard.rb

```diff
@@ -0,0 +1,18 @@
+module Claims
+  class AddSchoolWizard < BaseWizard
+    def define_steps
+      add_step(SchoolStep)
+      add_step(SchoolOptionsStep) if steps[:school].school.blank?
```

Comment:
Could this cause an undefined method error? Would `fetch` be more appropriate here?

Reply:
I've also addressed this in all other wizards